### ISSUE TITLE
##IMPORTANT## WEATHER ##IMPORTANT##

### ISSUE THREAD

**Issue by Rmdhn1** [whataudoinghere/Feed_and_Weather_Plugin#18]
http://4pda.ru/forum/index.php?showtopic=903913&st=3980#entry81861074

**Comment by rainabba** [comment]
Can you summarize what you wanted us to take/learn from that link?

**Comment by Rmdhn1** [comment]
oh huh i forgot about this, i think i remember cause it wasn't working with the latest version beta at the time i commented cause the beta on 4pda has pretty cool new features but not really working with this, i guess its not a beta anymore and probably implemented, closing this
